docs/News.md

```diff
@@ -1,3 +1,20 @@
+## Jan 4, 2022
+*   **Added data** consisting of 27,447 samples from 10 studies: 
+    * [Endometrial Carcinoma (CPTAC, Cell 2020)](https://www.cbioportal.org/study/summary?id=ucec_cptac_2020) *95 samples*
+    * [Pancreatic Ductal Adenocarcinoma (CPTAC, Cell 2021)](https://www.cbioportal.org/study/summary?id=paad_cptac_2021) *140 samples*
+    * [Lung Squamous Cell Carcinoma (CPTAC, Cell 2021)](https://www.cbioportal.org/study/summary?id=lusc_cptac_2021) *108 samples*
+    * [Lung Adenocarcinoma (CPTAC, Cell 2020)](https://www.cbioportal.org/study/summary?id=luad_cptac_2020) *110 samples*
+    * [Glioblastoma (CPTAC, Cell 2021)](https://www.cbioportal.org/study/summary?id=gbm_cptac_2021) *99 samples*
+    * [Breast Cancer (CPTAC, Cell 2020)](https://www.cbioportal.org/study/summary?id=brca_cptac_2020) *122 samples*
+    * [Pediatric Brain Cancer (CPTAC/CHOP, Cell 2020)](https://www.cbioportal.org/study/summary?id=brain_cptac_2020) *218 samples*
+    * [Metastatic Prostate Cancer (Provisional, June 2021)](https://www.cbioportal.org/study/summary?id=mpcproject_broad_2021) *123 samples*
+    * [MSK MetTropism (MSK, Cell 2021)](https://www.cbioportal.org/study/summary?id=msk_met_2021) *25,775 samples*
+    * [Cancer Therapy and Clonal Hematopoiesis (MSK, 2021)](https://www.cbioportal.org/study/summary?id=msk_ch_ped_2021) *657 samples*
+
+*   **Added TMB (nonsynonymous) scores** for all studies. [Example: new TMB field for study gbm_cptac_2021](https://www.cbioportal.org/study/clinicalData?id=gbm_cptac_2021)   
```

Comment:
One thing I just noticed Yichao is that the significant digits is not consistent across the computed TMB values (at least in the example).  Also is it necessary to have some many significant digits in the TMB value? A quick check of the impact clinical sample data file, it seems like the clinical impact tmb is rounded to 2 significant digits.  Maybe this was discussed already w @ritikakundra @jjgao, but I thought I would bring this up.

![image](https://user-images.githubusercontent.com/366003/148815597-85f844fb-e0ec-4a50-9b81-0eb18ac423f3.png)

Reply:
@n1zea144 asked front-end help to round it up when displaying the table. (cc' @jjgao @ritikakundra )